### DISCUSSION TITLE
Boot on USB Power without battery

### DISCUSSION THREAD

**Discussion by AlexanderHackl** [gopro/labs#491] [General]
Hi,
is there a way to boot the GoPro (I am using a Hero10 at the moment) on USB Power On without a battery in the camera?
I am controlling the GoPro via USB so i do not need the Trigger on Power Off feature.

Thank You!

**Comment by dnewman-gpsw** [comment]
Yes, that is the normal behavior of the !MWAKE=2 command. see https://gopro.github.io/labs/control/extensions/

**Reply by jxqqk** [reply]
I use the !MWAKE=2 with a boot script since january for a timelapse and I found it very realiable.
I had to factory reset the camera (HERO11) and I updated the labs firmware to 2.12.70 to try to fix the cloud service because is very unreliable.
Now wake command from USB power do not work anymore, tried reflashing, !reset, anything.
When powered, red led flash one time but the camera do not boot anymore.
Downgrading to not works, some error about battery (I have no battery only usb power).
Any suggestion? Thank you.

**Reply by dnewman-gpsw** [reply]
Sounds like you camera is not updating, or developed a fault. Try removing Labs, to make camera is working normally, use this firmware https://gopro.com/en/us/update/hero11-black  If that works, we can discuss Labs stuff here, it is not working, contact support for a camera replacement.

**Reply by jxqqk** [reply]
Thank you.
With a battery inserted, no problem to install latest normal firmware as suggested, camera works normally.
Then inserted lab firmware without problems, but WAKE do not works.
I tried but was not able to revert to 2.10.70.
GM

**Reply by dnewman-gpsw** [reply]
The wake is working on 2.12.70.  Try a different power source, also share your qrlog.txt file.

**Reply by jxqqk** [reply]
Thank you.
I tried battery and two different power sources very powerful (both worked in the past).
No messages before power on.
Here the log:
[2023-08-21 19:03:16-delayed] wake reason:POWER_BUTTON 
[2023-08-21 19:03:16-delayed] Labs:Display Messages 
[2023-08-21 19:03:19] Features activated: 
TUSB,WAKE=2,BOOT: 
[2023-08-21 19:03:21] Labs:Main Start 
[2023-08-21 19:03:21] Labs:H22.01.02.12.70 
[2023-08-21 19:03:21] Labs:Initialize 
[2023-08-21 19:03:22] Boot command:!Lparti 
[2023-08-21 19:03:23] Load Script:C:\MISC\parti
..........

There is a way to revert to previous firmware?
GM

**Reply by dnewman-gpsw** [reply]
I have confirmed what you are seeing.  2.12.00 (Main fW) came with some fixes for trusting USB, is seems to be conflicting with Labs. Having the battery inserted is the only solution. There is no rolling back to 2.10.xx at the moment, something in the main FW prevents that downgrade. I will be looking for a fix now for WAKE without a battery. Fortunately more Labs FW is coming.

**Reply by dnewman-gpsw** [reply]
@jxqqk Try this 2.12.71, patching this one error  https://www.dropbox.com/t/mHuJUQ6O60b9q4N2

**Reply by jxqqk** [reply]
Flawless update (with battery inserted) and WAKE works again when USB powered only(no battery).
Thank you

**Reply by dnewman-gpsw** [reply]
I will make sure this patch is within the next release.  Keep using 2.12.71 in the meantime.

**Reply by jxqqk** [reply]
Thank you again. Is it rock solid like it was before (not a problem for eight months) or do you suggest to wait for the final release?
I need to reput the camera on site unattended like it was before for many time (and hoping the cloud services now works better with new regular firmware...)

**Reply by dnewman-gpsw** [reply]
It should be solid, the "final" release will be based on newer firmware, and that is not ready yet.  2.12.71 is good.

**Comment by GPExpert** [comment]
@dnewman-gpsw Hi David. I have a HERO10 on the latest Labs firmware (v1.62.70).

I have exactly the same issue discussed in #491.

The camera is powered from a regulated 5V supply. With no battery installed, WAKE=2 does not cold boot when USB power is applied. The identical setup works with a battery installed. 

I noticed that 2.12.71 fixed this exact problem, but the Dropbox link has expired.

Has this fix definitely been carried into v1.62.70, or has there been a regression? If not, would you be willing to provide an updated test build? Thanks so much for all your legendary help!
